src/lib.rs

```diff
@@ -215,6 +219,28 @@ impl Client {
             .output_of(&task.with_model(model), how)
             .await
     }
+
+    pub async fn tokenize(
```

Comment:
A minimal example with a comment why you want to call this would be nice. Not required though for me to merge the PR

Reply:
ok, I added docstring examples for tokenize()/detokenize().